modules/oe_content_call_proposals/config/install/core.entity_form_display.node.oe_call_proposals.default.yml

```diff
@@ -0,0 +1,263 @@
+langcode: en
```

Comment:
The form configuration is missing all the default maxlength configuration values for the general fields: oe_content_short_title, oe_summary, oe_teaser and (I think, but better we double check this last one)

Reply:
I can't find the `oe_teaser` field in the `Call for proposals` content type. I added the `maxlength` configuration to the `oe_content_short_title` and `oe_summary`  fields.